SocialFilter/sections/specific.txt

```diff
@@ -48,6 +48,7 @@ japanese.joins.com,sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.d
 ! ##.content__post--meta-social
 otomobilgunluklerim.com,tilki.net,ekosayfa.com,bilisimogretmeni.com,azbuz.org,siberkalem.com,teknobird.com,buneymis.net,teknoxhaber.com,fosforludusunceler.com,kirtkirtla.com,oyuncukuru.org,denizcilikbilgileri.com,hknkr.com,kadioglublog.com##.content__post--meta-social
 !
+oricon.co.jp##.sns-share-area2  
```

Comment:
Better `oricon.co.jp#$#div[class^="sns-share-"] > a[onclick^="socialAct"] { visibility: hidden !important; }`.
Will fix it directly.

Reply:
thank you.